fs/xfstests.py

```diff
@@ -78,14 +78,16 @@ def setUp(self):
                              'libattr-devel', 'libaio-devel', 'libuuid-devel',
                              'openssl-devel', 'xfsprogs-devel'])
 
-            if self.detected_distro.name == 'rhel' and\
-                    self.detected_distro.version.startswith('8'):
-                packages.remove('indent')
             if self.detected_distro.name == 'SuSE':
                 packages.extend(['libbtrfs-devel', 'libcap-progs'])
             else:
                 packages.extend(['btrfs-progs-devel'])
 
+            if self.detected_distro.name == 'rhel' and\
```

Comment:
Why moving the lines ?

Reply:
Package is getting added here: https://github.com/avocado-framework-tests/avocado-misc-tests/blob/master/fs/xfstests.py#L84

Reply:
Ah ok, missed the removal of btrfs...